module-apps/application-phonebook/windows/PhonebookContactOptions.hpp

```diff
@@ -17,6 +17,7 @@ namespace gui
         auto handleSwitchData(SwitchData *data) -> bool override;
 
       private:
+        bool externalRequest = false;
```

Comment:
lets create enum from it with two values -> it will be easier to read in future than bool.

Reply:
done